### PR TITLE
build(editorconfig): set indent_size to 4 for python files

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -10,6 +10,9 @@ insert_final_newline = true
 [*.{c,h,in,lua}]
 max_line_length = 100
 
+[*.py]
+indent_size = 4
```
Comment (gpanders) [inline .editorconfig:14]: Want to go ahead and include `indent_style = space` too?

File: .editorconfig [hunk 2]
```diff
@@ -10,6 +10,10 @@ insert_final_newline = true
 [*.{c,h,in,lua}]
 max_line_length = 100
 
+[*.py]
+indent_style = space
```
Comment (zeertzjq) [inline .editorconfig:14]: Isn't this already specfied above?

Comment (gpanders) [reply .editorconfig:14]: Yea I guess it’s already included in the wildcard section. Doesn’t hurt to be explicit imo but I don’t care either way.

@dundargoc it’s up to you

**Review by gpanders — APPROVED** [review]

File: .editorconfig (hunk 2 above)
Comment (dundargoc) [reply .editorconfig]: shipit

**Pull request merged**